src/main/connector/system/Windows/Windows.yaml

```diff
@@ -204,19 +204,49 @@ monitors:
               BytesSentPerSec,
               BytesReceivedPerSec
             FROM Win32_PerfRawData_Tcpip_NetworkInterface
+          computes:
+            - type: duplicateColumn
+              column: 1
+            - type: awk
+              script: 'BEGIN {FS=OFS=";"} {gsub(/[^a-zA-Z0-9]/, " ", $2); print $0}'
+        # Name;IpAddress
+        networkAdapters:
+          type: wmi
+          namespace: root\CIMv2
+          query: >
+            SELECT Description,
+              IPAddress
+            FROM Win32_NetworkAdapterConfiguration
+          computes:
+            - type: duplicateColumn
+              column: 1
+            - type: awk
+              script: 'BEGIN {FS=OFS=";"} {gsub(/[^a-zA-Z0-9]/, " ", $2); print $0}'
+            - type: duplicateColumn
+              column: 3
+            - type: awk

```

Comment:
This awk script can be replaced with a simple `Replace` compute step:

```yaml
- replace: "|"
- replaceBy: " "
- column: 4
```

(or you could combine all 4 compute steps into a single AWK script)